### PR TITLE
data: BSatishSutar

### PR DESCRIPTION
**Pull request by BSatishSutar**

### CODE REVIEW

File: data/BSatishSutar.json
Comment (ChinmayMhatre) [inline data/BSatishSutar.json:9]:
```suggestion
      "icon": "FaLinkedin"
```

File: data/BSatishSutar.json
Comment (ChinmayMhatre) [inline data/BSatishSutar.json:24]:
```suggestion
      "icon": "FaGitlab"
```

File: data/BSatishSutar.json
Comment (ChinmayMhatre) [inline data/BSatishSutar.json:19]:
```suggestion
      "icon": "FaGithub"
```

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
Looks good, Just a few changes to the icon name.

**Review by iamthecloverly — APPROVED** [review]

**Review by ChinmayMhatre — CHANGES_REQUESTED** [review]
The icons names are still incorrect, could you take a look at the suggestion in the previous review?

File: data/BSatishSutar.json
Comment (BSatishSutar) [reply data/BSatishSutar.json]: Chinmay,

I am unable to find that JSON file to make corrections.
Please advise!
I have checked in branch I have created in my repo and Eddie's repo too...
I am learning GitHub so kindly guide me.
Thanks Chinmay,

Comment (BSatishSutar) [reply data/BSatishSutar.json]: Hey Chinmay, I found the file finally and I have made corrections as suggested. Thank you!

**Review by SurajPratap10 — APPROVED** [review]
looks good

**Comment by BSatishSutar** [comment]
> looks good

Thanks, Suraj!

Could you please let me know when and how will I get notified about the approval?

**Comment by SurajPratap10** [comment]
> > looks good
> 
> Thanks, Suraj!
> 
> Could you please let me know when and how will I get notified about the approval?

Whenever the maintainers will be free, they will merge and you will get it through mail that ur PR has been merged.

**Pull request merged**
